manim/config/utils.py

```diff
@@ -137,6 +137,10 @@ def make_config(parser):
     )
     config["frame_y_radius"] = config["frame_height"] / 2
     config["frame_x_radius"] = config["frame_width"] / 2
+    config["small_buff"] = constants.SMALL_BUFF
+    config["med_small_buff"] = constants.MED_SMALL_BUFF
+    config["med_large_buff"] = constants.MED_LARGE_BUFF
+    config["large_buff"] = constants.LARGE_BUFF
```

Comment:
Issue: Instead of adding constants to the config dict, modules that need to access these values should access them through the constants module, not the config. For example, if a module was reading `config["small_buff"]`, we should change it to read from `constants.SMALL_BUFF` (and not add small_buff to config).

Reply:
I'll remove this and instead use direct access to the constants within VideoSeries' class... although based on the above feedback I'm uncertain whether drawings.py will continue to exist.